### ISSUE TITLE
[BUG] Player Count isnt shown on bots status

### ISSUE THREAD

**Issue by Zenlet** [Exiled-Team/DiscordIntegration#94]
**Describe the bug**
Bot does not display the amount of people on the server no matter how long is waited

**To Reproduce**
1. Join the server
2. Look at discord
3. Look at bot
4. Feast your eyes

**Expected behavior**
Bot to display the amount of players on the server

**EXILED Version ("latest" is not a version):**
5.1.1

**Comment by ScuutScuutLucas** [comment]
Can confirm this is happening, this are my logs:

```
10:21:50 Discord     Discord.Net v3.5.0 (API v9)
10:21:50 Gateway     Connecting
10:21:51 Gateway     You're using the GuildScheduledEvents gateway intent without listening to any events related to that intent, consider removing the intent from your config.
10:21:51 Gateway     You're using the GuildInvites gateway intent without listening to any events related to that intent, consider removing the intent from your config.
10:21:51 Gateway     Connected
10:21:52 Gateway     Ready
10:22:00 OnReceived  Object reference not set to an instance of an object.
10:22:00 CreateBasic Sending embed Server 1 Logs.
10:22:10 OnReceived  Object reference not set to an instance of an object.
10:22:20 OnReceived  Object reference not set to an instance of an object.
```

**Comment by WoofWolfGG** [comment]
> Can confirm this is happening, this are my logs:
> 
> ```
> 10:21:50 Discord     Discord.Net v3.5.0 (API v9)
> 10:21:50 Gateway     Connecting
> 10:21:51 Gateway     You're using the GuildScheduledEvents gateway intent without listening to any events related to that intent, consider removing the intent from your config.
> 10:21:51 Gateway     You're using the GuildInvites gateway intent without listening to any events related to that intent, consider removing the intent from your config.
> 10:21:51 Gateway     Connected
> 10:21:52 Gateway     Ready
> 10:22:00 OnReceived  Object reference not set to an instance of an object.
> 10:22:00 CreateBasic Sending embed Server 1 Logs.
> 10:22:10 OnReceived  Object reference not set to an instance of an object.
> 10:22:20 OnReceived  Object reference not set to an instance of an object.
> ```

I have an identical situation

**Comment by joker-119** [comment]
The logs you posted are unrelated to this bug, and are caused by you having invalid channel IDs in your logs config.

Status bug should be fixed in 1.4.0 for real this time
